ChangeLog

```diff
@@ -1,3 +1,29 @@
+Change highlights in uncrustify-0.68 (November 2018)
+-------------------------------------------------------------------------------
+
+New options:
+  - nl_class_leave_one_liner_groups      Oct 17
+  - nl_inside_namespace                  Aug 26
+  - sp_return_brace                      Aug 21
+  - align_assign_decl_func               Jul 24
+  - sp_brace_brace                       Jul 23
+  - sp_after_decltype                    Jul 22
+  - align_right_cmt_same_level           Jul 14
+  - nl_oc_mdef_brace                     Jul  8
+  - mod_enum_last_comma                  Jul  8
+  - sp_type_ellipsis                     Jun 15
+  - indent_continue_class_head           May 30
+  - sp_decltype_paren
```

Comment:
I think this is my bad, but there are a couple options (`sp_paren_ellipsis`, `sp_sizeof_ellipsis`, `sp_sizeof_ellipsis_paren`) that got added under 0.67 that actually should be here (apparently).

BTW, no date for `sp_decltype_paren`? (Should be Jul 21: 0f34dcb5d4ed... or Jul 24 if you go by merge date?)

Reply:
Thanks.

Reply:
> There are a couple options that got added under 0.67 that actually should be here (apparently).

Nudge. LGTM otherwise.